testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -426,6 +426,8 @@
 PFncdf_Top = (_data_ref / 'Amber/posfor.top').as_posix()
 PFncdf_Trj = (_data_ref / 'Amber/posfor.ncdf').as_posix()
 
+PRMmulti = (_data_ref / "Amber/multi_ache.prmtop").as_posix()
```

Comment:
PRMmulti doesn't really give me a sense of what is in this file. Can we either rename it to "PRM_chainids" or at the very least have a comment explaining what this file contains?

Reply:
Done